app/src/main/java/com/mobileproto/hireddit/hireddit/speech/WordToSpeech.java

```diff
@@ -41,6 +41,12 @@ public void speak(String toSpeak)
         }
     }
 
+    public void pause() {
+        if (mTtobj.isSpeaking()) {
+            mTtobj.stop();
+        }
+    }
+
     public void stop()
```

Comment:
Okay, that a makes sense. Naming it pause is a bit confusing to me though because it's also stoping/destroying, and there's no way to resume